src/parser/SparqlParserHelpers.cpp

```diff
@@ -122,4 +122,25 @@ std::string ParserAndVisitor::unescapeUnicodeSequences(std::string input) {
   output += view.substr(lastPos);
   return output;
 }
+
+// _____________________________________________________________________________
+bool isValidVariableName(std::string_view var) {
+  sparqlParserHelpers::ParserAndVisitor parserAndVisitor{std::string{var}};
+  try {
+    auto [result, remaining] =
+        parserAndVisitor.parseTypesafe(&SparqlAutomaticParser::var);
+    return remaining.empty();
+  } catch (...) {
+    return false;
+  }
+}
+
+// As soon as a program is linked against this file, the following
+// initialization has the effect that `Variable::isValidVariableName` performs
+// as expected (See `Variable.h` for details).
+[[maybe_unused]] const static int isValidVarSetterDummy = []() {
+  Variable::isValidVariableName() = &isValidVariableName;
+  return 42;
+}();
+
```

Comment:
Any chance this fixed the issue I encountered in https://github.com/ad-freiburg/qlever/pull/2128/files#diff-98c2cc1de8639467ca369fc9b6dac31d86475e5870def0fcf9a22ee6f7a7f642R632-R635 ?